code/client/makepkginfo

```diff
@@ -385,6 +385,15 @@ def check_mode(option, opt, value, parser):
                                                             (opt, mode))
     setattr(parser.values, option.dest, value)
 
+def has_valid_install_critieria(catinfo):
```

Comment:
That would probably work; return True if installer_type == 'profile'. (for bonus points also could make sure PayloadIdentifier exists, but I can't imagine makepkginfo generating a plist without that...)